k8s/spinnaker/scripts/install/setup.sh

```diff
@@ -20,7 +20,7 @@ NUM_ENABLED_APIS=$(gcloud services list --project $PROJECT_ID \
 
 if [ $NUM_ENABLED_APIS != $NUM_REQUIRED_APIS ]; then
   bold "Enabling required APIs ($REQUIRED_APIS)..."
-  bold "This will take a few minutes (progress will not be reported during this operation)..."
+  bold "This may take 10 minutes or more (progress will not be reported during this operation)..."
```

Comment:
The expected duration here refers to enabling the required apis. The "(progress will not be reported during this operation)" bit was added because following the apis being enabled, various progress updates are in fact streamed to the terminal.

I don't ever recall this phase taking 10 minutes. Just timed one and it was 2.5 minutes.

I'm guessing you're referring to the overall installation time?

If so, I don't think this is the right spot to indicate this.

Reply:
Maybe we can change it to something like:

This phase will take a few minutes (progress will not be reported during this operation).

Once the required APIs are enabled, the remaining components will be installed and configured. The entire installation may take 10 minutes or more...

Reply:
Paging @dorbin for an opinoin.

Reply:
LGTM, though I'd end with a `.`, not a `...` :)